.coveragerc

```diff
@@ -1,8 +1,8 @@
 [run]
 branch = True
 parallel = True
-include=$TOP/synapse/*
-data_file = $TOP/.coverage
+include=$GITHUB_WORKSPACE/synapse/*
```

Comment:
Was a bit sad to tie this file to GHA rather than something agnostic---but no worse than before?